lib/block-patterns/three-images-gallery.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Three columns with offset images.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Three columns with offset images', 'Block pattern title', 'default' ),
+	'categories'  => array( 'gallery' ),
+	'content'     => '<!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"25%"} -->
+	<div class="wp-block-column" style="flex-basis:25%"><!-- wp:image {"id":240,"sizeSlug":"large","linkDestination":"none","className":"is-style-default"} -->
+	<figure class="wp-block-image size-large is-style-default"><img src="https://s.w.org/images/core/5.8/architecture-01.jpg" alt="' . esc_attr__( 'Close-up, abstract view of geometric architecture.', 'default' ) . '" class="wp-image-240"/></figure>
+	<!-- /wp:image --></div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column {"width":"25%"} -->
+	<div class="wp-block-column" style="flex-basis:25%"><!-- wp:spacer {"height":500} -->
+	<div style="height:500px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+	
+	<!-- wp:spacer {"height":150} -->
+	<div style="height:150px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
```

Comment:
The combination of these two spacers stacked results in a bit of awkward space on mobile:

<img width="329" alt="Screen Shot 2021-04-13 at 2 24 31 PM" src="https://user-images.githubusercontent.com/5375500/114604077-5c4e9200-9c4d-11eb-81f3-cb0a2dfde059.png">

Reply:
Yeah, I think that's a broader spacer issue, and would need to be resolved by #10081 or #22956.